### DISCUSSION TITLE
Is it possible to pre-render all translations in Nuxt 3 using SSG mode?

### DISCUSSION THREAD

**Discussion by StefanVDWeide** [nuxt-modules/i18n#1811] [Q&A]
I'm currently in the process of migrating my website from Nuxt 2 to Nuxt 3 and I was wondering if there is a way to pre-render all the translations while using the `nuxi generate` command.

At the moment when generating the pages, all `.HTML` files contain the translation keys so when loading the page, all keys will be visible to the visitor and only after the browser is finished parsing, the correct text is shown. This is of course not every good for UX.

Is there any way around this?

**Comment by Dari4sho** [comment]
I'm not sure what I did set up exactly, but for me they are already translated in the generated files, so I would suggest checking it again (almost a year later).
